tools/generate_og_images.py

```
@@ -0,0 +1,58 @@
+    text = content_type.capitalize()
+    text_width, text_height = draw.textsize(text, font=font)
+    text_x = (width - text_width) / 2
+    text_y = (height - text_height) / 2
+
+    draw.text((text_x, text_y), text, font=font, fill=text_color)
+
+    output_path = os.path.join(output_dir, f"{content_type}_og_image.png")
+    image.save(output_path)
+    print(f"OG image saved to {output_path}")
+
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)